app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -280,6 +288,49 @@ export class EcsServerGroupConfigurationService {
       });
   }
 
+  public setCapacityProviderDetails(command: IEcsServerGroupCommand): void {
+    this.$q.all({
+      capacityProviderDetails: this.ecsClusterReader.describeClusters(command.credentials, command.region)
+    }).then((result: Partial<IEcsServerGroupCommandBackingData>) => {
+      command.backingData.capacityProviderDetails = chain(result.capacityProviderDetails)
+        .map((cluster) => this.mapCapacityProviderDetails(cluster))
+        .value();
+    });
+
+    if(command.ecsClusterName != null || command.ecsClusterName.length > 0){
```

Comment:
Should this be an && condition? (do we want to configure available capacity if it's an empty string?

Reply:
Ohhh yes. I will change this.